src/Components/Admin/Admin.js

```diff
@@ -9,13 +10,35 @@ import AdminDashboard from "./AdminDashboard";
  * Author: Harsha Srikara
  * Date: 3/22/20
  */
+
+var firebaseConfig = {
```

Comment:
Can we move this configuration into a firebaseconfig.json file and read from that?